pemFioi/quickAlgo/subtask.js

```diff
@@ -176,6 +179,16 @@ var initBlocklySubTask = function(subTask, language) {
       updateScores();
    };
 
+   subTask.reloadFunctions = function(newFunctions) {

```

Comment:
This function is better split in two : the first line (the one setting the function list) is unrelated to the rest of the function.

I'd rename this function `subTask.reloadLevel` and then move the first line to another function, or directly before where this function was called.